### DISCUSSION TITLE
Mirth Connect 4.4.0

### DISCUSSION THREAD

**Discussion by jdonextgen** [nextgenhealthcare/connect#5864] [General]
Mirth Connect 4.4.0 is a major release containing new features like adding new functionality to the Mirth Connect Setup Wizard such as: 
1. Adding the ability to download and install your Mirth Connect commercial extensions at the time Mirth Connect is installed or upgraded.
2. When upgrading Mirth Connect, the Setup Wizard now displays the existing values for the Destination Directory, License Key, Network Ports, and Password Requirements from your mirth.properties file rather than the standard default values or the values that were entered previously.

Other new features include improved event log messages and a new server event that fires when the Mirth Connect Server has finished starting up. This release also contains enhancements for the Mirth Connect Administrator Launcher, the Mirth Connect Docker images, and several bug fixes, security improvements, and updates to both Mirth Connect Core and our commercial extensions. 

[See What's New](https://github.com/nextgenhealthcare/connect/wiki/4.4.0---What's-New) | [Upgrade Guide](https://github.com/nextgenhealthcare/connect/wiki/Upgrading) | [Release Notes](https://github.com/nextgenhealthcare/connect/milestone/77?closed=1)

# Downloads

### Connect

OS | Arch | Download Link | SHA256 | MD5
-- | ---- | ------------- | ------ | ---
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-windows-x32.exe) | `e94677be0714a67f1cabd470a1b83276a66d7150747c51203947686968a9a8e1` | `adc3345680eb06c772b22453fae86237`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-windows-x32.zip) | `7f67f4a5861aae7c165880881890cb7c1b41f8ec5be8ce60a695f08d186a7b43` | `27d18156c483d44adf1482c718e6a28a`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-windows-x64.exe) | `34223057e0f01a2ce100197e0c3d3d908723df882a2196f683610daa00eef8c0` | `9a74373f6c5ece81af43c788751a9c8a`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-windows-x64.zip) | `a38f6720b1e1455f6deda597777c66510bc688d548af681e5de4620ce754416d` | `0852d97e35b28c6c6fceec055d1fc9fc`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [RPM](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-linux.rpm) | `41ffa1f1f5faa956514361d53e2f25233ebfbf8b0456374f45a51e26a830d180` | `b83ba6d8c33230811965eea11e0f8a1e`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-unix.tar.gz) | `17ceead28e02a4327ceffcdd9c2bafdba43fc7d3c78d5be537d1e792874de210` | `5cdf304f90d495d75737a524f99f3ad8`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-unix.sh) | `5557a47345e5186f19295f3c8b188e50533da9196e7670103d696b8e9731d9d2` | `f41de0d1b3e224dc9ad37d7fd59b93d6`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Installer](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-macos.dmg) | `f397ae88b947d09b0b124c0379c543ae94aeb9131f2f07b19066844d72a36149` | `200a26525fd3d6f2951fdc7f288bab68`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnect-4.4.0.b2948-unix.tar.gz) | `17ceead28e02a4327ceffcdd9c2bafdba43fc7d3c78d5be537d1e792874de210` | `5cdf304f90d495d75737a524f99f3ad8`

### Command Line Interface

OS | Arch | Download Link | SHA256 | MD5
-- | ---- | ------------- | ------ | ---
![Windows](https://i.imgur.com/ZgvUEYg.png) | x86 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnectcli-4.4.0.b2948-windows-x32.zip) | `e9cbebed000f558abe2f5b5a244db60044a5ad5e6e452c6a212df67831ad0b99` | `617e0e376b8f70d5f8b31269dd34c7ab`
![Windows](https://i.imgur.com/ZgvUEYg.png) | x64 | [Zip](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnectcli-4.4.0.b2948-windows-x64.zip) | `c69f9c0b66e0c9915c2146c11c865de474b22f021c86c3a0df1594b80fe5d8d8` | `1ffb8fdd729e95d343435aedc09ec0db`
![Linux](https://i.imgur.com/ZAdY18p.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnectcli-4.4.0.b2948-unix.tar.gz) | `dbf9c42c1a97acbfc5dda109d86c8c140fcd771b71b406db6f00506fd61b4a1e` | `bca09d36213667a296e79a80d3385c37`
![Mac OS X](https://i.imgur.com/8hXbSsA.png) | x64 | [Tar.gz](https://s3.amazonaws.com/downloads.mirthcorp.com/connect/4.4.0.b2948/mirthconnectcli-4.4.0.b2948-unix.tar.gz) | `dbf9c42c1a97acbfc5dda109d86c8c140fcd771b71b406db6f00506fd61b4a1e` | `bca09d36213667a296e79a80d3385c37`

<hr /><em>This discussion was created from the release <a href='https://github.com/nextgenhealthcare/connect/releases/tag/4.4.0'>Mirth Connect 4.4.0</a>.</em>

**Comment by antoinetielbeke** [comment]
Where can I find additional information for:
> We added the ability to download and install your Mirth Connect commercial extensions at the time Mirth Connect is installed or upgraded. This simplifies the process so that users no longer have to download commercial extensions prior to the installation.

**Reply by mlarcelet** [reply]
This is a nice feature. Does anyone know which URLs are called while downloading the extensions ? (need to explicitely allow this outgoing traffic from our servers)
